src/main/java/fruitshop/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,18 @@
+package fruitshop.strategy;
+
+import fruitshop.db.Storage;
```

Comment:
According to the checklist, don't keep all logic in a single package. Consider organizing your classes into appropriate packages.